pkg/config/tlscfg/options_test.go

```diff
@@ -231,3 +231,68 @@ func TestToOtelClientConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestToOtelServerConfig(t *testing.T) {
+	testCases := []struct {
+		name     string
+		options  Options
+		expected configtls.ServerConfig
+	}{
+		{
+			name: "default mapping",
+			options: Options{
+				ClientCAPath: "path/to/client/ca.pem",
+				CAPath:       "path/to/ca.pem",
+				CertPath:     "path/to/cert.pem",
+				KeyPath:      "path/to/key.pem",
+				CipherSuites: []string{"TLS_RSA_WITH_AES_128_CBC_SHA"},
+				MinVersion:   "1.2",
+				MaxVersion:   "1.3",
+			},
+			expected: configtls.ServerConfig{
+				ClientCAFile: "path/to/client/ca.pem",
+				Config: configtls.Config{
+					CAFile:       "path/to/ca.pem",
+					CertFile:     "path/to/cert.pem",
+					KeyFile:      "path/to/key.pem",
+					CipherSuites: []string{"TLS_RSA_WITH_AES_128_CBC_SHA"},
+					MinVersion:   "1.2",
+					MaxVersion:   "1.3",
+				},
+			},
+		},
+		{
+			name: "overrides set",
```

Comment:
what do you mean by overrides?

Reply:
This was to test this if condition. Let me know if a better test name works. 
```go
	if o.ReloadInterval > 0 {
		cfg.ReloadClientCAFile = true
	}
```

Reply:
```suggestion
			name: "with reload interval",
```